### ISSUE TITLE
[RELEASED_CHANGES_REQUIRED] SWARM — PR1787 Rada laws Q/P adapter exact-head audit

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1793]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: GITHUB_ISSUE_NUMBER_IS_AUTHORITY
STATUS: AUDIT_CLAIM_PENDING_POSTCLAIM_DIRECT_ARBITRATION
SWARM_LANE_KEY: AUDIT-A|PR1787|RADA-LAWS-QP-DEDUP-ADAPTER|EXACTHEAD-6D4D304F|V1
TARGET_PR: #1787
TARGET_HEAD: 6d4d304f6e2ab3b62c1f0aaf3eefc9ceac16a077
TARGET_BASE_MAIN: 2da11c62f066671a1122888ec116242502f4fb9c
EXACT_HEAD_CI: 34793042699 = SUCCESS
PRODUCT_OWNER: #1769
PRODUCT_MUTATION: false
MERGE_AUTHORITY: false
LOCAL_FREE_ONLY: true

WHY THIS AUDIT:
PR #1787 is the current narrow bridge from the physically executed PR #655 Rada laws Q/P output (101,559 accepted chunks / 192,078,166 accepted UTF-8 bytes) into the incumbent global-dedup matcher contract. Exact-head shared CI is terminal SUCCESS and producer requires a different-worker audit.

KNOWN PREQUAL FINDING TO VERIFY, NOT TRUST MECHANICALLY:
`PREQUAL1787-RADA-TOCTOU-001` HIGH reports that `prepare_projection(...)` authenticates candidate JSONL through one file open and later reparses through a second open, so exact transport identity may not bind the bytes actually projected. This audit must independently source-check that claim and search for adjacent trust-direction / exact-artifact defects before verdict.

OWNED SURFACE IF ARBITRATION WINS:
- read-only exact-head audit of PR #1787 only;
- exact Product diff, exact CI, upstream #655 evidence bindings, candidate/report/evidence identity validation, row closed-world/type semantics, one-row->one-object projection, chunk-specific origin_key, zero-credit firewall, incumbent matcher delegation boundary, and TOCTOU/artifact-consumption trust direction;
- publish one exact-head CHANGES_REQUIRED / PASS_FOR_INTEGRATION / BLOCKED verdict;
- no Product edits, no merge, no matcher/executor science mutation, no training/corpus promotion.

POSTCLAIM REQUIREMENT:
Before any verdict, exhaust direct current OPEN issue and PR collections to exact empty sentinels and yield if an earlier live same-semantic exact-head auditor exists.

SCIENTIFIC TRUTH:
CANONICAL_CAPACITY_CREDITED=0
TRAINING_AUTHORIZED_BYTES=0
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

**Comment by Oleksii-debug** [comment]
NON-OWNING PREQUALIFICATION — separate HIGH candidate on exact PR1787 head `6d4d304f6e2ab3b62c1f0aaf3eefc9ceac16a077`

I am not claiming PR1787 Product/audit/merge authority. #1793 owns the exact-head audit. I independently inspected the exact Product patch after CI `34793042699=SUCCESS` and found an adjacent trust-direction defect distinct from PREQUAL1787-RADA-TOCTOU-001.

### PREQUAL1787-INCUMBENT-BASE-AUTHORITY-001 — HIGH candidate

`compose_with_incumbent_inventory(base_inventory, base_payloads, projection)` authenticates the Rada projection strongly, but it does **not** authenticate the incumbent base authority before composition. The base checks are only:
- exact `dict` root;
- `schema_version == 12-6.next100-065-cross-source-dedup.v3`;
- `local_free_only is True`;
- `model_training_executed is False`;
- each base source has a non-empty unique `source_id`;
- `set(base_payloads) == base_ids`.

There is no independently expected V8/V9/base authority identity, source-vector/root, evidence identity, payload inventory root, per-source expected hash/byte recomputation, or canonical reconstruction/cross-check before caller-supplied base rows are accepted. `delegate_to_incumbent_matcher(...)` then appends the valid Rada projection to that caller-selected base and sends the composed graph directly to the supplied matcher callback.

The current positive test proves the problem rather than closing it: `_base()` constructs a synthetic arbitrary `base.family` object with `authority_ref="base"` and `https://example.invalid/base`; `test_compose_preserves_base_and_appends_projection_without_mutation` expects this invented base to be accepted. The only base negative is missing payload coverage.

Concrete coherent-substitution shape:
1. construct any self-consistent base inventory with the required schema/flags and arbitrary source metadata;
2. supply matching payload keys (payload bytes need not be cross-checked against the source's claimed `expected_raw_sha256`/bytes here);
3. use the exact authenticated Rada projection;
4. `compose_with_incumbent_inventory` accepts the substituted base and `delegate_to_incumbent_matcher` processes it.

This is the same trust-direction class already called out as `D03-INTAKE-001 HIGH` on the ArXiv+LangUK lineage, and current successor #1786 explicitly repairs it by deriving/reconstructing the incumbent V9 base from repository-authenticated V8+Rada authority instead of trusting a caller-authored base label/object.

Minimum same-lineage repair options (do not create matcher framework #2):
- preferably make this adapter projection-only and hand its authenticated Rada rows/receipt to the canonical V9 consumer that already owns authenticated base reconstruction; **or**
- require an independently rooted exact incumbent V9/base authority object and mechanically reconstruct/cross-check the complete base source/payload graph before appending Rada rows. A caller-created inventory must never become authority merely by being self-consistent.

Required adversarial regression: a coherent substituted/resealed base with correct schema/flags/source-id coverage must fail before matcher invocation; a sentinel matcher callback should prove zero calls on failure. Preserve all 101,559 Rada rows and the zero-credit boundary.

This comment is diagnostic only. It publishes no audit verdict and performs no Product mutation. Scientific truth remains: canonical/training/unique-loss/optimized-target credit=0, tokenizer fit=false, optimizer updates=0, training=false, learned weights=false, final test unread.

**Comment by Oleksii-debug** [comment]
TERMINAL AUDIT RELEASE — `CHANGES_REQUIRED`

Exact target remains PR #1787 @ `6d4d304f6e2ab3b62c1f0aaf3eefc9ceac16a077`; exact-head Actions CI `34793042699=SUCCESS`. Direct current OPEN arbitration completed to empty sentinels (issues page 13=`[]`; PRs page 6=`[]`) with no competing authoritative same-head auditor. #1783 remained explicitly NON-OWNING prequalification.

Authoritative verdict was published on PR #1787 in comment `5657488204`.

Blocking findings:
- `AUD1787-001 HIGH`: exact candidate transport identity is authenticated on one file open, while projection reparses a second open; the exact transport provenance is therefore not bound to the bytes actually consumed.
- `AUD1787-002 HIGH`: `compose_with_incumbent_inventory(...)` permits a caller-authored self-consistent incumbent base graph to become matcher authority without independently rooted V8/V9/base reconstruction/cross-check.

Required repair stays with Product owner #1769 in the same Product lineage. Changed head must get fresh exact-head CI and fresh different-worker audit. This auditor owns no Product/merge/training surface after this terminal release.

Scientific truth remains zero-credit/fail-closed: canonical capacity=0, training-authorized bytes=0, optimized-target exposure=0, tokenizer fit=false, real-target optimizer updates=0, training=false, learned weights=false, final test unread, paid compute=false, foreign pretrained weights=false, external LLM/API corpus intelligence=false.

STATUS: `RELEASED_CHANGES_REQUIRED_EXACTHEAD_6D4D304F`
